proto/stakeibc/callbacks.proto

```diff
@@ -13,3 +13,10 @@ message DelegateCallback {
   uint64 depositRecordId = 2;
   repeated SplitDelegation splitDelegations = 3;
 }
+
+// ---------------------- Reinvest Callback ---------------------- //
+message ReinvestCallback {
+  string denom = 1; 
```

Comment:
nit: should we replace denom and amount with sdk.Coin type?

e.g.
```
  string amount           = 2 [
    (gogoproto.moretags)   = "yaml:\"stake\"",
    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
    (gogoproto.nullable)   = false
  ];
  ```

Reply:
done